internal/cli/ips.go

```diff
@@ -0,0 +1,103 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ipAddress = Argument{
+		Name: "IP",
+		Help: "IP address to check",
+	}
+)
+
+func ipsCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "ips",
+		Short:   "Manage blocked IP addresses",
+		Long:    "Manage blocked IP addresses.",
+		Aliases: []string{"ip"},
```

Comment:
Do we need this alias, since `ips` is already short?

Reply:
Yeah, probably not, i'll remove.